### PR TITLE
Update global-json.md

### PR DESCRIPTION
**Pull request by mairaw**
global.json reference was missing entries and examples.

Pending questions:

Does the packages property work? Or should we add a note about this dotnet/cli#3816?
What happens if runtime is not specified?
What happens if architecture is not specified?

/cc @blackdwarf @livarcocc

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.runtime.compilerservices.asynctaskmethodbuilder-1?branch=pr-en-us-996)

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.diagnostics.eventtypefilter?branch=pr-en-us-996)

**Comment by blackdwarf** [comment]
@mairaw if the runtime and the architecture are not set, it will default to `coreclr` and `x64`. In all honesty, I haven't see these being set on many `global.json` files.

File: docs/core/tools/global-json.md [hunk 1]
```diff
@@ -14,30 +14,75 @@ ms.assetid: e1ac9659-425f-4486-a376-c12ca942ead8
...
 
+```json
+{
+    "sdk": {
+        "version": "1.0.0-preview2-003121"
+    }
+}
+```
 
+### architecture
```
Comment (BillWagner) [inline docs/core/tools/global-json.md]: Is this valid on non-Windows platforms? Or is it simply ignored?

Comment (mairaw) [inline docs/core/tools/global-json.md]: Good question @BillWagner. @blackdwarf?

**Comment by mairaw** [comment]
Even if they are not used much, I think it's important to have them all documented @blackdwarf. Right?

However, I was reading online that the default architecture depends on which runtime you configured. Not sure if this still holds true, for example: https://github.com/aspnet/dnvm/issues/468

File: docs/core/tools/global-json.md (hunk 1 above)
Comment (blackdwarf) [inline docs/core/tools/global-json.md]: I don't know, really.

**Comment by livarcocc** [comment]
@gkhanna79 Can you confirm the SDK options? I know that version is correct. Not sure about architecture though.

**Comment by gkhanna79** [comment]
The host does not own the SDK options - tools (e.g. CLI) own setting it up. The only configuration of SDK that host is aware of "version" that it uses to determine which SDK to load CLI from.

**Comment by mairaw** [comment]
So, should I leave the other options out of the document then, @gkhanna79?
These were the options listed in the [global.json schema](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/global.json) and was trying to determine their default values.

**Comment by mairaw** [comment]
@gkhanna79 can you help with my previous question or know someone who can? Thanks.

**Comment by gkhanna79** [comment]
@livarcocc should be able to help. @schellap may have some ideas.

**Comment by schellap** [comment]
`"/sdk/version"` this json path gets the value of the SDK version to use to compile the app etc.

I'm not sure about the other two options. Runtime and architecture.

**Comment by saldana** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json?branch=pr-en-us-996](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json?branch=pr-en-us-996)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C3%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611030008102237-996%5Cworkflow_report.html).

**Comment by mairaw** [comment]
I've removed the controversial properties. 

Is the topic accurate from what you guys know about global.json now? 

/cc @blackdwarf @gkhanna79 @schellap

**Comment by saldana** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json?branch=pr-en-us-996](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json?branch=pr-en-us-996)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C3%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611030025505986-996%5Cworkflow_report.html).

**Comment by mairaw** [comment]
I'm merging this and deal with any feedback later if this is inaccurate.

**Comment by johnsimons** [comment]
So is this suppose to be merged and released ?
Because https://docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json still not up to date!

**Comment by mairaw** [comment]
@johnsimons yes! What do you see missing?

**Comment by johnsimons** [comment]
There is no doco for "projects" and "packages", all I see is the "sdk".

image

**Comment by johnsimons** [comment]
So it looks like it was removed in https://github.com/dotnet/docs/commit/c2722327c3ca213309cae5e0d0417b0f0e9ea514#diff-250d1d2a2050498b0df08adc4609d9c0

**Comment by livarcocc** [comment]
projects and packages are no longer relevant in msbuild world. The only value in global.json that is still honored is the sdk version.

**Comment by johnsimons** [comment]
@livarcocc thanks for the clarification

**Pull request merged**